shell/packages/sandstorm-accounts-ui/login_buttons.js

```diff
@@ -2,6 +2,9 @@
 var loginButtonsSession = Accounts._loginButtonsSession;
 
 var helpers = {
+  currentRoute: function (routeName) {
```

Comment:
Please name helpers that return booleans `isCondition`, to help make its behavior in the context where it's being called.

For instance, I'd expect a function named `currentRoute` to return `Router.current().route` as a string.